contracts/CertificateStore.sol

```diff
@@ -1,19 +1,41 @@
-pragma solidity ^0.4.4;
+pragma solidity ^0.4.17;
 
+import "zeppelin-solidity/contracts/ownership/HasNoEther.sol";
 
-contract CertificateStore {
+
+contract CertificateStore is HasNoEther {
   address public issuer;
```

Comment:
HasNoEther also extends the Ownable stuff too so this contract has an owner that's in addition to issuer